src/components/Login/LoginContainer.tsx

```diff
@@ -89,6 +89,7 @@ export const InitialisationContainer: FunctionComponent<NavigationProps> = ({
   const resetStage = (): void => {
     setLoginStage("SCAN");
   };
+  const [lastResendWarningMessage, setLastResendWarningMessage] = useState("");
```

Comment:
can you elaborate on why we need the use of state hook to set the resend error message?

Reply:
let me try using something else!